### DISCUSSION TITLE
feature: scratchpad

### DISCUSSION THREAD

**Discussion by octvs** [glzr-io/glazewm#634] [Q&A]
Hey there,

Searching "scratchpad" through the repository via github gui yields no results [^1].
However this is a feature [^2] I make great use of on i3/sway. Closes I get is to
minimize a window and recall it via `Alt-Tab` however the recall doesn't work
from another workspace so makes it a bit pointless. Is this possible or ever
offered here before?

Cheers

[^2]: https://i3wm.org/docs/userguide.html#_scratchpad
[^1]: https://github.com/search?q=repo%3Aglzr-io%2Fglazewm%20scratchpad&type=code

**Comment by bitFlow66** [comment]
I would also appreciate this feature a lot.

**Reply by DinDotDout** [reply]
Yup, I usually use bindings to toggle a specific app on and off. I have a bad workaround using ahk but it works pretty poorly and requires glaze not to manage the windows. On other TWMs it usually either moves the scratchpad with you to any workspace when jumping around or hides them when unfocused.
By allowing bindings to target specific apps or somehow storing the pids when first launched this could kind of be implemented by setting a keybind to move the specific app to a named workspace ans back. Other functionality like the one i mentioned before (autohide or move) would be a plus too ofc
